gsa/src/gmp/models/cpe.js

```diff
@@ -30,7 +30,8 @@ class Cpe extends Info {
   static parseElement(element) {
     const ret = super.parseElement(element, 'cpe');
 
-    ret.severity = parseSeverity(ret.max_cvss);
+    ret.severity = parseSeverity(ret.score / 10);
+    delete ret.score;
     delete ret.max_cvss;
```

Comment:
I don't think max_cvss is even in the response any more, is it? In that case we do not need this line any more.